packages/mongodb-chatbot-ui/src/Message.tsx

```diff
@@ -153,14 +134,14 @@ const markdownProps = {
     },
     ol: ({ children, ordered, ...props }) => {
       return (
-        <Body as="ol" className={styles.markdown_list} {...props}>
+        <Body as="ol" {...props}>
```

Comment:
should `styles.markdown_list` be deleted accordingly?

Reply:
Yeah it already is. These styles got merged into `markdown_container`.